packages/apps/immutable-records/package.json

```diff
@@ -39,6 +47,8 @@
     "@vanilla-extract/next-plugin": "2.3.1",
     "eslint": "^8.45.0",
     "eslint-config-next": "13.4.5",
+    "lokijs": "^1.5.12",
```

Comment:
What is this dependency being used for?

Reply:
`lokijs` and `pino-pretty` were both required by `@walletconnect/modal` and `@walletconnect/sign-client`, it gives build errors when I did not add these, seems like a bug on their side. It's dev-dependencies anyway, I don't think its a big deal right now.